[2025-06-03T09:32:37.524+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: usd_rate_fetcher_taskflow.save_data_to_db scheduled__2000-03-26T00:00:00+00:00 [queued]>
[2025-06-03T09:32:37.536+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: usd_rate_fetcher_taskflow.save_data_to_db scheduled__2000-03-26T00:00:00+00:00 [queued]>
[2025-06-03T09:32:37.536+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 3
[2025-06-03T09:32:37.559+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): save_data_to_db> on 2000-03-26 00:00:00+00:00
[2025-06-03T09:32:37.568+0000] {standard_task_runner.py:57} INFO - Started process 4976 to run task
[2025-06-03T09:32:37.572+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'usd_rate_fetcher_taskflow', 'save_data_to_db', 'scheduled__2000-03-26T00:00:00+00:00', '--job-id', '7645', '--raw', '--subdir', 'DAGS_FOLDER/usd_rate_fetcher.py', '--cfg-path', '/tmp/tmpm5cgj0iz']
[2025-06-03T09:32:37.574+0000] {standard_task_runner.py:85} INFO - Job 7645: Subtask save_data_to_db
[2025-06-03T09:32:37.644+0000] {task_command.py:410} INFO - Running <TaskInstance: usd_rate_fetcher_taskflow.save_data_to_db scheduled__2000-03-26T00:00:00+00:00 [running]> on host 40869fde016f
[2025-06-03T09:32:37.869+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-06-03T09:32:37.962+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='usd_rate_fetcher_taskflow' AIRFLOW_CTX_TASK_ID='save_data_to_db' AIRFLOW_CTX_EXECUTION_DATE='2000-03-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2000-03-26T00:00:00+00:00'
[2025-06-03T09:32:37.965+0000] {usd_rate_fetcher.py:51} INFO - Received USD rate: 28.33 for date 2000-03-26
[2025-06-03T09:32:37.983+0000] {base.py:73} INFO - Using connection ID 'pg_database' for task execution.
[2025-06-03T09:32:37.990+0000] {usd_rate_fetcher.py:62} WARNING - Rate already exists for date 2000-03-26
[2025-06-03T09:32:37.992+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-06-03T09:32:38.000+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=usd_rate_fetcher_taskflow, task_id=save_data_to_db, execution_date=20000326T000000, start_date=20250603T093237, end_date=20250603T093238
[2025-06-03T09:32:38.034+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-06-03T09:32:38.055+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
